api-reference/_hidden/GridBaseColumn/allowReordering.md

```diff
@@ -5,9 +5,11 @@ default: true
 ---
 ---
 ##### shortDescription
-Specifies whether this column can be used in column reordering at runtime. Applies only if [allowColumnReordering](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/allowColumnReordering.md '{basewidgetpath}/Configuration/#allowColumnReordering') is **true**.
+Specifies whether this column can be reordered at runtime.
```

Comment:
We can get rid of passive voice and convey the thought from the note right here, in the short description:

```suggestion
Specifies whether users can reorder this column. Overrides the [allowColumnReordering](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/allowColumnReordering.md '{basewidgetpath}/Configuration/#allowColumnReordering') property value.
```